### ISSUE TITLE
Apply new styleguide - Ensure accessibility

### ISSUE THREAD

**Issue by jenweber** [ember-learn/cli-guides#180]
For applying the new styleguide, before we ship, we need to evaluate the accessibility and fix problems uncovered.

This is the branch that has the WIP work:
https://github.com/ember-learn/cli-guides/pull/141

This accessibility review should be done once we feel like most of the visual bugs are knocked out on that branch.
